### ISSUE TITLE
Why are all my excess bytes getting elided?

### ISSUE THREAD

**Issue by robsimmons** [autolab/Autolab#160]
Autolab suddenly stopped showing my students their feedback:

```
Autograder [Thu Jan 22 02:20:01 2015]: Success: Autodriver returned normally
Autograder [Thu Jan 22 02:20:01 2015]: Here is the output from the autograder:
---
Autodriver: Job exited with status 0
Thu Jan 22 01:19:47 EST 2015
[15-122] Extraction...
=============================================
scavhunt.c0
puzzle.c0

Thu Jan 22 01:19:47 EST 2015
[15-122] Compilation...
=============================================
Thu Jan 22 01:19:48 EST 2015
[15-122
...[excess bytes elided]...
Test puzzletest same_start_end, expect successful execution
Test passed!
==================================================

*** FINISHED!
*** TOTAL SCORE:  10
{"scores": {"piazza": 1, "puzzle": 5, "handout": 1, "afsimg": 1, "tutorial": 1, "feedback": 1}}
```

that bit in the middle is information I'd intended to show them!

**Comment by robsimmons** [comment]
This seems to now be resolved via email,etc, so I'm closing.

**Comment by niveditc** [comment]
This issue seems to be surfacing again for the next assignment of 15-122 :

```
Autodriver: Job exited with status 0
Fri Jan 23 21:31:25 EST 2015
[15-122] Extraction...
==================================================
pixel.c0
pixel-bad.c0
pixel-test.c0
tasks.c0

Fri Jan 23 21:31:26 EST 2015
[15-122] Compilation...
==================================================
pix
...[excess bytes elided]...
nd
Task 2: 2/2
Task 3: 0/3
Task 4: 0/1
Task 5: 0/2
Task 6: 0/2
Task 7: 0/2

*** FINISHED!
*** TOTAL SCORE: 4
{"scoreboard": [1, 0, 1], "scores": {"badpixel": 2, "testpixel": 0, "testquantize": 0, "quantize": 0, "countzero": 0, "removered": 0, "pixel": 2}}
```

**Comment by niveditc** [comment]
Regrading it seemed to fix the issue, and I'm not sure why. As a preventive measure, can we increase the max output size even more for our course? This was the entire output :

```
Autograder [Fri Jan 23 22:55:53 2015]: Received job 15122-s15_pixels_3_xxxxxx@andrew.cmu.edu:308
Autograder [Fri Jan 23 22:56:02 2015]: Success: Autodriver returned normally
Autograder [Fri Jan 23 22:56:02 2015]: Here is the output from the autograder:
---
Autodriver: Job exited with status 0
Fri Jan 23 22:55:40 EST 2015
[15-122] Extraction...
==================================================
pixel.c0
pixel-bad.c0
pixel-test.c0
tasks.c0

Fri Jan 23 22:55:41 EST 2015
[15-122] Compilation...
==================================================
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
pixel-test.c0:31.3-31.24:error:too few arguments in function call
  make_pixel(0xF23C45D);
  ~~~~~~~~~~~~~~~~~~~~~ 
Compilation failed
Fri Jan 23 22:55:43 EST 2015
[15-122] Compiling tasks.c0 with different pixel
         implementations. If you didn't respect
         the pixel.c0 interface, there will be
         error messages here and they won't make
         much sense...
==================================================
tasks.c0:28.23-28.24:error:type mismatch
expected: int
   found: pixel
  return 0xFF00FFFF & p;
                      ~ 
Compilation failed
tasks.c0:28.23-28.24:error:type mismatch
expected: int
   found: pixel
  return 0xFF00FFFF & p;
                      ~ 
Compilation failed
Fri Jan 23 22:55:44 EST 2015
[15-122] Grading...
=============================================
TESTING...
==================================================
Testing Task 1 (pixel.c0)
--------------------------------------------------
Checking for the existence of pixel-grade - passed.
--------------------------------------------------
Test pixel-grade get_component, expect successful execution - passed.
--------------------------------------------------
Test pixel-grade make_pixel, expect successful execution - passed.
--------------------------------------------------
Test pixel-grade test_pixel_interface, expect successful execution - passed.
--------------------------------------------------
Test pixel-grade make_pixel_toobig1, expect assertion failure - passed.
Test pixel-grade make_pixel_toobig2, expect assertion failure - passed.
Test pixel-grade make_pixel_toobig3, expect assertion failure - passed.
Test pixel-grade make_pixel_toobig4, expect assertion failure - passed.
Test pixel-grade make_pixel_toosmall1, expect assertion failure - passed.
Test pixel-grade make_pixel_toosmall2, expect assertion failure - passed.
Test pixel-grade make_pixel_toosmall3, expect assertion failure - passed.
Test pixel-grade make_pixel_toosmall4, expect assertion failure - passed.
==================================================
Testing Task 2 (pixel-bad.c0)
--------------------------------------------------
Checking for the existence of pixel-failure - passed.
--------------------------------------------------
Checking for the existence of pixel-exploit - passed.
--------------------------------------------------
Test pixel-exploit, expect assertion failure - passed.
--------------------------------------------------
Test pixel-failure, expect assertion failure - passed.
==================================================
Testing Task 3 (pixel-test.c0)
--------------------------------------------------
Checking for the existence of pixeltest-self 
*** TEST FAILED! Expected the file to exist and it did not.
*** Hint: Your pixel.c0 does not compile against your pixel-test.c0
==================================================
Testing Task 4 (tasks.c0 - remove_red)
--------------------------------------------------
Test tasks-grade remove_red, expect successful execution - passed.
--------------------------------------------------
Test tasks-int remove_red, expect successful execution 
*** TEST FAILED! Expected successful execution.
*** Actual outcome: The program, when run, produced the wrong answer
*** Hint: Some part of tasks.c0 doesn't seem to respect the pixel interface.
*** (The error messages above may give you a hint as to where)
==================================================
Testing Task 5 (tasks.c0 - quantize)
--------------------------------------------------
Test tasks-grade quantize, expect successful execution 
*** TEST FAILED! Expected successful execution.
*** Actual outcome: The program, when run, produced the wrong answer
*** Hint: Our tests for quantize fail
==================================================
Testing Task 6 (tasks.c0 - test_quantize)
--------------------------------------------------
Test tq-good test_quantize, expect successful execution - passed.
--------------------------------------------------
Test tq-self test_quantize, expect successful execution 
*** TEST FAILED! Expected successful execution.
*** Actual outcome: An assertion failed unexpectedly
*** Hint: Your quantize tests did not succeed when run on your own solution (no penalty, but ugh!)
--------------------------------------------------
Test tq-bad1 test_quantize, expect assertion failure 
*** TEST FAILED! Expected a contract to fail.
*** Actual outcome: Your program ran to completion, but should have signaled an error or failed a contract
*** Hint: Didn't catch bug 1 (returns wrong answers)
--------------------------------------------------
Test tq-bad2 test_quantize, expect assertion failure - passed.
--------------------------------------------------
Test tq-bad3 test_quantize, expect assertion failure 
*** TEST FAILED! Expected a contract to fail.
*** Actual outcome: Your program ran to completion, but should have signaled an error or failed a contract
*** Hint: Didn't catch bug 3 (returns wrong answers)
--------------------------------------------------
Test tq-bad4 test_quantize, expect assertion failure 
*** TEST FAILED! Expected a contract to fail.
*** Actual outcome: Your program ran to completion, but should have signaled an error or failed a contract
*** Hint: Didn't catch bug 4 (edge case)
--------------------------------------------------
Test tq-bad5 test_quantize, expect assertion failure 
*** TEST FAILED! Expected a contract to fail.
*** Actual outcome: Your program ran to completion, but should have signaled an error or failed a contract
*** Hint: Didn't catch bug 5 (edge case)
==================================================
Testing Task 7 (tasks.c0 - count_zeroes)
--------------------------------------------------
Test tasks-grade count_zeroes, expect successful execution 
*** TEST FAILED! Expected successful execution.
*** Actual outcome: The program, when run, produced the wrong answer
*** Hint: Tests for count_zeroes fail
==================================================
Task 1: 2/2 + 1 point that will be graded by hand
Task 2: 2/2
Task 3: 0/3
Task 4: 0/1
Task 5: 0/2
Task 6: 0/2
Task 7: 0/2

*** FINISHED!
*** TOTAL SCORE: 4
{"scoreboard": [1, 0, 1], "scores": {"badpixel": 2, "testpixel": 0, "testquantize": 0, "quantize": 0, "countzero": 0, "removered": 0, "pixel": 2}}
```

**Comment by dlbucci** [comment]
@mihirpandya is currently working on Tango trying to fix #169.  I believe that resetting may undo the autodriver settings changes, so this might start happening, but it should only be temporary.
